models/ideas.js

```diff
@@ -102,9 +84,22 @@ function getAllIdeas() {
     return ideasWithDonations;
 }
 
+function getIdea(ideaId) {
+    const ideas = getAllIdeas();
+
+    const idea = ideas.find((idea) => ideaId === idea.id);
+
+    if (!idea) {
+        throw new Error('Error. No idea with such ID found.');
+    }
+
+    return idea;
+}
+
 module.exports = {
-    createIdea,
+    createNewIdea: createIdea,
```

Comment:
Can call the key and the value the same here: `createIdea`?

Reply:
Renamed.